CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### [0.3.0] 2018-12-19
```

Comment:
Please use `### [0.3.0] NEXT` because there is an automation process here 👍

Reply:
done